src/prf_expand.rs

```diff
@@ -108,3 +108,20 @@ impl PrfExpand<([u8; 96], [u8; 32], [u8; 4])> {
     pub const SAPLING_ZIP32_CHILD_NON_HARDENED: Self = Self::new(0x12);
 }
 with_inputs!(a, A, b, B, c, C);
+
+impl PrfExpand<([u8; 32], [u8; 4])> {
+    pub const REGISTERED_ZIP32_CHILD: Self = Self::new(0xAC);
```

Comment:
Shouldn't this constant be on line 99? Why a separate `impl` block for the same type?

Reply:
It is a coincidence that it is the same type. The `with_tag` method can't be expressed via the `with_inputs!` macro (because that macro only supports fixed-length `u8` arrays), so we need a manual implementation of `with_tag`, and that implementation is logically associated only with `REGISTERED_ZIP32_CHILD` (even though, because of an implementation detail, it ends up being available but unused for the other cases of `PrfExpand<([u8; 32], [u8; 4])>` as well).